h2o-r/tests/testdir_parser/runit_parse_encrypted_zip.R

```diff
@@ -0,0 +1,30 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+# Tests parsing of encrypted zip file data 
+
+test.parseEncryptedZip <- function() {
+  crimes_path_zip <- locate("smalldata/chicago/chicagoCrimes10k.csv.zip")
+  crimes_path_zip_aes <- file.path(sandbox(), "crimes.csv.zip.aes")
+
+  keystore_path <- locate("smalldata/extdata/keystore.jks")
+  keystore <- h2o.importFile(keystore_path, parse = FALSE)
+
+  decrypt_tool <- h2o.decryptionSetup(keystore, key_alias = "secretKeyAlias",
+                                      password = "Password123", cipher = "AES/ECB/PKCS5Padding")
+
+  args <- c(keystore_path, "JCEKS", "secretKeyAlias", "Password123", "AES/ECB/PKCS5Padding", 
+            crimes_path_zip, crimes_path_zip_aes)
+  tool_result <- h2o.rapids(sprintf('(run_tool "EncryptionTool" ["%s"])', paste(args, collapse = '", "')))
```

Comment:
What is the purpose of EncryptionTool in this test? it doesn't seem to be involved in `h2o.importFile(crimes_path_zip_aes, decrypt_tool = decrypt_tool)`

Reply:
This is a way of encrypting the zip file on the fly when the test runs. Basically, I added a simple tool to encrypt files and I am invoking it through a rapid expression. That way the process is self-documenting as opposed to storing an encrypted file on S3 in smalldata.

Reply:
ok, got it. Thanks for your eplanation!